py/server/deephaven/json/__init__.py

```diff
@@ -0,0 +1,1221 @@
+    _build(
+        builder,
+        allow_missing,
+        allow_null,
+        allow_int=True,
+        allow_decimal=True,
+        allow_string=allow_string,
+    )
+    if on_missing:
+        builder.onMissing(dtypes.BigDecimal(str(on_missing)))
+    if on_null:
+        builder.onNull(dtypes.BigDecimal(str(on_null)))
+    return JsonValue(builder.build())
+
+
+def any_val() -> JsonValue:
+    """Creates an "any" value. The resulting type is implementation dependant.
+
+    Returns:
+        the "any" value
+    """
+    return JsonValue(_JAnyValue.of())
+
+
+def skip_val(
+    allow_missing: Optional[bool] = None,
+    allow_null: Optional[bool] = None,
+    allow_int: Optional[bool] = None,
+    allow_decimal: Optional[bool] = None,
+    allow_string: Optional[bool] = None,
+    allow_bool: Optional[bool] = None,
+    allow_object: Optional[bool] = None,
+    allow_array: Optional[bool] = None,
+    allow_by_default: bool = True,
+) -> JsonValue:
+    """Creates a "skip" value. No resulting type will be returned, but the JSON types will be validated as configured.
+    This may be useful in combination with an object type where allow_unknown_fields=False. For example, the JSON object
+
+    .. code-block:: json
+        { "name": "foo", "age": 42 }
+
+    might be modelled as the object type
+
+    .. code-block:: python
+        object_val({ "name": str, "age": skip_val() }, allow_unknown_fields=False)
+
+    Args:
+        allow_missing (Optional[bool]): if a missing JSON value is allowed, by default is None
+        allow_null (Optional[bool]): if a JSON null type is allowed, by default is None
+        allow_int (Optional[bool]): if a JSON integer type is allowed, by default is None
+        allow_decimal (Optional[bool]): if a JSON decimal type is allowed, by default is None
+        allow_string (Optional[bool]): if a JSON string type is allowed, by default is None
+        allow_bool (Optional[bool]): if a JSON boolean type is allowed, by default is None
+        allow_object (Optional[bool]): if a JSON object type is allowed, by default is None
+        allow_array (Optional[bool]): if a JSON array type is allowed, by default is None
+        allow_by_default (bool): the default behavior for the other arguments when they are set to None, by default is True
+
+    Returns:
+        the "skip" value
+    """
+
+    def _allow(x: Optional[bool]) -> bool:
+        return x if x is not None else allow_by_default
+
+    builder = _JSkipValue.builder()
+    _build(
+        builder,
+        allow_missing=_allow(allow_missing),
+        allow_null=_allow(allow_null),
+        allow_int=_allow(allow_int),
+        allow_decimal=_allow(allow_decimal),
+        allow_string=_allow(allow_string),
+        allow_bool=_allow(allow_bool),
+        allow_object=_allow(allow_object),
+        allow_array=_allow(allow_array),
+    )
+    return JsonValue(builder.build())
+
+
+def json_val(json_value_type: JsonValueType) -> JsonValue:
```

Comment:
This would make sense at the top of the file since a number of functions above it use it.